Rulebook 3.0 (#113)
International-Data-Spaces-Association/IDSA-Rulebook: 006_DSGA.md

# Dataspace Governance Authority

## Definition

> A Dataspace Governance Authority (DSGA) is a functional role. It is responsible for providing the basic governance framework, defining which [Dataspace Trust Frameworks (DTFs)](009_Dataspace_Trust_Frameworks.md) are being used for the data space, potentially defining an explicit DTF by defining business processes, policies and semantic models to be used by participants in the dataspace. 

Reviewer: ```suggestion
The Data Space Governance Authority (DSGA) represents the collective governance function of a data space. It is responsible for providing the basic governance framework, defining which [Dataspace Trust Frameworks (DTFs)](009_Dataspace_Trust_Frameworks.md) are being used for the data space, potentially defining an explicit DTF by defining business processes, policies and semantic models to be used by participants in the dataspace. 
```

Author: functional role is an important terminus that should remain. collective governance function is imprecise as this could include business governance, which the DSGA does not provide.

The DSGA ensures interoperability through minimal shared semantics.

> The DSGA is NOT the legal entity which provides runtime-enforcement. It is the collection of DTFs used within the dataspace and any additional provisions, definitions and processes specific to the individual data space.

Reviewer: ```suggestion
The DSGA may be established as a legal entity that enables participants to take binding collective decisions, including the definition of membership criteria, business processes, policies, and the semantic models to be used within the data space. In this role, the DSGA provides the institutional framework for decision-making and evolution of the data space.

The DSGA does not perform runtime enforcement. Technical enforcement of policies and usage conditions is carried out by the respective participants and their technical components.

From a normative perspective, the DSGA is expressed through the set of Data Space Trust Frameworks (DTFs) adopted by the data space, together with any additional provisions, definitions, and processes specific to that data space. These collectively define the binding rules under which participants operate.
```

Author: I disagree with your edits. The DSGA is a functional role and never an entity. This is one of the big problems and misunderstandings. A dataspace organization is not the DSGA, it may support the functional implementation of a DSGA, but they are clearly separate concepts. One is a logical thing within the definition of a dataspace, the other one is an implementation detail in the physical world. two completely separate elements. It is important to understand that a DSGA exists even if there is no dataspace organization.

## Implementation of the DSGA

- **By the Participants**: each participant has full knowledge of the DSGA and can enforce the rules of the DSGA in interactions with other participants. Value-Added Services or external Oracles might provide additional information needed to implement the governance model defined by the DSGA.

- **Service Providers**: multiple service providers operate value added data space services or external Oracles that enable the enforcment of the governance model provided by the DSGA. The participants of the data space are responsible to choose which services providers to use and are free to reject interactions with participants where a common set of accepted service providers cannot be identified. An example of such a service can be the issuance of a data space participant credential.

> - **Operations Company**: in a very centralized designed data space the DSGA functions can be implemented by a single operations company which provides mandatory governance enforcement services. Note, that this is the least recommended design as it creates centralized control and single points of failure, thus impacting the autonomy and agency of participants.

Reviewer: ```suggestion
- **Operations Company**: in a centrally governed data space the DSGA functions can be implemented by a single operations company which provides mandatory governance enforcement services. Note, that this creates centralized control and single points of failure, thus impacting the autonomy and agency of participants.
```

## Additional considerations for DSGA design

- **Minimal Semantics**: Interoperability is achieved through shared core concepts (e.g., participant identity attributes, data space operations), not comprehensive ontologies (e.g. semantic models of the data being shared in the data space) to reduce complexity.

> - **Socio-Technical Coupling**: Governance rules need to integrate technical feasibility with legal participant agreements. Ambiguities in policy interpretation are resolved through defined rules in the DSGA or its referenced DTFs. DSGA can provide prodcedures and escalation paths, assuming reasonable consensus thresholds to resolve conflicts in the data space.

Reviewer: ```suggestion
- **Socio-Technical Coupling**: Governance rules need to integrate technical feasibility with legal participant agreements. Ambiguities in policy interpretation are resolved through defined rules in the DSGA or its referenced DTFs. DSGA can provide procedures and escalation paths, assuming reasonable consensus thresholds to resolve conflicts in the data space.
```
